frontend/src/metabase/setup/containers/PostSetupApp.jsx

```diff
@@ -143,7 +144,8 @@ export default class PostSetupApp extends Component {
                     <div className="mb1">
                       <Quotes quotes={QUOTES} period={2000} />
                     </div>
-                    <ThinProgressBar />
+                    {/*The percentage is hardcoded so we can animate this*/}
+                    <ProgressBar isAnimated="true" percentage=".5" />
```

Comment:
@kdoh I think I'm doing something wrong here because it seems like the conditional isn't flipping true, and hence I'm not seeing any animation.